README.md

```diff
@@ -1,37 +1,60 @@
-9. Run the site locally with `yarn develop`. The site should now be running at `http://localhost:8000/`!
+## MacOS Installation
+We recommend using MacOS to work with the EDB Docs application.
 
-## Running Without a Local Installation (Recommended for Windows)
+1. Install [Homebrew](https://brew.sh/), if it's not already installed. (Use `brew -v` to check.)
 
-If you wish to work with Docs without installing the prerequesites locally, you can do so from within a Docker container using VSCode. See: [Working on Docs in a Docker container using VSCode](README_DOCKER_VSCODE.md)
+1. Install Git using Homebrew with `brew install git`, if it's not already installed.  (Use `git --version` to check.)
+
+1. Clone the [EDB Docs GitHub repo](https://github.com/EnterpriseDB/docs) to your local machine using [GitHub Desktop](https://desktop.github.com/), or via the Terminal with `git clone https://github.com/EnterpriseDB/docs.git`
+
+1. Set up an SSH key in GitHub, if you haven't done so already. (Go to [GitHub's SSH Keys page](https://github.com/settings/keys) to check.) If you don't have an SSH Key set up yet, you'll need to set one up to authenticate you to GitHub. See [GitHub's SSH docs](https://docs.github.com/en/github-ae@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for more information.
+
+1. Install [Node.js version 14 LTS](https://nodejs.org/en/download/). We recommend using Node version 14 LTS (the Long Term Support release) as version 15 is not compatible with some of our dependencies at this time.
+
+    - If you already have Node installed, you can verify your version by running `node -v` in the cloned repo directory.
+
+    - If you already have Node version 15 installed, you may want to consider using Node Version Manager (NVM) for a simpler way to manage multiple versions of Node.js. Follow the [directions to install NVM](https://github.com/nvm-sh/nvm#installing-and-updating), then run `nvm install` in the cloned repo directory, followed by `nvm use 14` to use version 14 LTS.
+
+1. Install Python 3 with `brew install python3`, if it's not already installed. (Use `python3 -V` to check that you have version 3.6 or higher.) Python is not needed for the core Gatsby system, but is required by several source scripts.
+
+1. Navigate to the cloned repo directory in your Terminal, if you haven't already done so.
+
+1. Install Yarn with `npm i -g yarn`. Yarn is the package manager we're using for this project, instead of NPM.
+
+1. Install Gatsby with `npm i -g gatsby-cli`. Gatsby is the software that powers the EDB Docs site.
+
+1. Install all required packages by running `yarn`.
+
+1. Pull the shared icon files down with `git submodule update --init`
+
+1. Select which sources you want with `yarn config-sources`, and then running `yarn pull-sources` if prompted (see section below for details)
```

Comment:
I believe explicitly running yarn pull-sources is no longer necessary (@epbarger ?) - might benefit more here from a bit of mildly prescriptive guidance, such as 
```suggestion
1. Select which sources you want with `yarn config-sources` - this will configure the products that will be built, and automatically pull them down. To speed up the next steps, choose only the products you intend to test or edit - you can change these options at any time (see below for details)
```

Reply:
Yeah the wording isn't quite right, `yarn config-sources` will launch right into `pull-sources` if needed, but you don't need to explicitly run `pull-sources` unless you're manually futzing around